packages/app/src/providers/h5grove/__snapshots__/h5grove-api.test.ts.snap

```diff
@@ -1431,6 +1431,11 @@ exports[`test file matches snapshot 1`] = `
               "size": 64,
             },
             "bool": {
+              "base": {
+                "class": "Integer",
+                "endianness": "little-endian",
+                "size": 8,
+              },
```

Comment:
Enums have a base type. I need to know what that is in order to use the right typed array constructor. It was already returned by h5grove, just not parsed.

Technically, it's probably always going to be `int8` with h5grove/h5py, but I prefer to be strict about it.